addons/arsenal/XEH_postInit.sqf

```diff
@@ -106,8 +108,9 @@ if (["ACE_dogtags"] call EFUNC(common,isModLoaded)) then {
                     private _dogtagData = [];
                     private _index = _allDogtags find _data;
                     _dogtagData = _allDogtagDatas select _index;
+                    private _dogtagString =  [localize ELSTRING(dogtags,itemName), ": ", (_dogtagData select 0)] joinString "";
```

Comment:
convert to function in dogtags pbo

Reply:
You don't even need to do that. Just do:
```sqf
private _dogtagString = [_data, ": ", dogtagData select 0] joinString "";
```

Even better would be to put those hooks into their respective components. That way you will easily also be able to see how those API functions can be improved to not need defines from the component directly (as that's pretty bad for any 3rd party mod using it, basically have to use IDCs directly or redefine those macros).

Reply:
I simply cannot ditch IDCs for those, it's impossible.

Reply:
data is the className in this case @jonpas, i'd have to use lnbText to retrieve the string.

Reply:
?
Just do something like that: `private _dogtagString =  [dogtagData selec 0] call EFUNC(dogtag,getFriendlyName);`

and put the content of the assignment into a function in dogtags `fn_getFriendlyName.sqf`

Reply:
I moved the EH to dogtags.